src/to-string.cpp

```diff
@@ -3,682 +3,608 @@
 
 #include "types.h"
 
-#define STRX( X ) make_less_screamy( #X )
```

Comment:
So before `get_string()` returned lower case works and now it's capital?
Are we sure it will not affect the wrappers? matlab/ unity......

Reply:
It returned capitalized: `ABC_DEF` -> `Abc Def`